src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -499,6 +285,913 @@ string SparqlQleverVisitor::visitTypesafe(
   return _prefixMap[prefix];
 }
 
+// ____________________________________________________________________________________
+void Visitor::visitTypesafe(Parser::BaseDeclContext* ctx) {
+  _prefixMap[""] = visitTypesafe(ctx->iriref());
+}
+
+// ____________________________________________________________________________________
```

Comment:
What is the rationale behind moving these implementations inside the file?
This makes it a little bit hard to keep track of the changes.

Reply:
I think that you suggested some time ago that it might be a good idea to move the code to the cpp. I tried to apply that somewhat consistently here and not have to many additional changes. Thats why the other changes will be in a separate PR. We can discuss in person whether/how I misinterpreted you comment.